cli.js

```diff
@@ -37,6 +37,7 @@ const cli = meow(`
 	  --debug                  Show the browser window to see what it's doing
 	  --overwrite              Overwrite the destination file if it exists
 	  --launch-options         Puppeteer launch options as JSON
+	  --clip                   An object which specifies clipping region of the page.
```

Comment:
Doesn't make sense to use JSON here. I'd rather see just `--clip="10 30 300 1024".

Reply:
And needs a better description.